app/controllers/answers/delete.js

```diff
@@ -0,0 +1,31 @@
+import { DEFAULT_ERROR_MESSAGE } from '~/utils/backend/constants';
+import { deleteAnswerSchema } from '~/utils/backend/validators/answer';
+import { db } from '~/utils/db.server';
+
+export const deleteAnswer = async (query) => {
+  const { error, value } = deleteAnswerSchema.validate(query);
+  const { answer_id } = value;
+
+  if (error) {
+    return {
+      errors: [{ message: DEFAULT_ERROR_MESSAGE, detail: error.details }],
```

Comment:
I think we should have the same consistency when we return errors.

I saw this return in another method or I don't know if I missing something 😅 

```
{
   error: {
      message: "msg",
      detail: "details"
   }
}
```

Reply:
Both `error` and `errors` are supported, to have single error or a list of errors if needed. Not sure if we should change to only just one though 😅